android/app/src/main/java/poke/rogue/helper/presentation/home/HomeActivity.kt

```diff
@@ -1,12 +1,67 @@
 package poke.rogue.helper.presentation.home
 
+import android.content.Intent
+import android.net.Uri
 import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
 import poke.rogue.helper.R
 import poke.rogue.helper.databinding.ActivityHomeBinding
 import poke.rogue.helper.presentation.base.BindingActivity
+import poke.rogue.helper.presentation.util.context.stringOf
+import poke.rogue.helper.presentation.util.context.toast
 
 class HomeActivity : BindingActivity<ActivityHomeBinding>(R.layout.activity_home) {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        setSupportActionBar(binding.toolbarHome.toolbar)
+        supportActionBar?.let { actionBar ->
+            actionBar.setDisplayHomeAsUpEnabled(true)
+            actionBar.setDisplayShowTitleEnabled(false)
+        }
+
+        initClickListeners()
+    }
+
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.menu_toolbar, menu)
+        return super.onCreateOptionsMenu(menu)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        when (item.itemId) {
+            R.id.it_toolbar_pokerogue -> {
+                toast(stringOf(R.string.toolbar_pokerogue))
```

Comment:
```suggestion
                toast(R.string.toolbar_pokerogue)
```

stringOf() 를 사용하지 않아도 됩니당 ㅎ ㅎ